employee-rostering-backend/src/main/java/org/optaweb/employeerostering/domain/Skill.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaweb.employeerostering.domain;
+
+import java.util.Objects;
+
+public class Skill extends AbstractPersistable {
+
+    private final Integer tenantId;
```

Comment:
Not Needed; tenantId in `AbstractPersistable`

Reply:
Yup, I'll fix that.